tests/environment_var/versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```

Comment:
shouldnt this be bumped to 0.12.whateverminimum?

Reply:
updated